index.js

```diff
@@ -0,0 +1,167 @@
+var instance_skel  = require('../../instance_skel');
+const WebSocket    = require('ws');
+var objectPath     = require('object-path');
+
+class instance extends instance_skel {
+	constructor(system, id, config) {
+		super(system, id, config);
+
+		this.ws = null;
```

Comment:
I'd recommend not setting `this.ws = null;` and either not declaring it or doing `this.ws = undefined;` (which is slightly stupid IMO)